services/src/main/java/org/keycloak/authentication/authenticators/x509/AbstractX509ClientCertificateAuthenticatorFactory.java

```diff
@@ -155,6 +157,12 @@
         ocspResponderUri.setLabel("OCSP Responder Uri");
         ocspResponderUri.setHelpText("Clients use OCSP Responder Uri to check certificate revocation status.");
 
+        ProviderConfigProperty ocspResponderCert = new ProviderConfigProperty();
+        ocspResponderCert.setType(TEXT_TYPE);
+        ocspResponderCert.setName(OCSPRESPONDER_CERTIFICATE);
+        ocspResponderCert.setLabel("OCSP Responder Certificate");
+        ocspResponderCert.setHelpText("Certificate used by the responder to sign the responses. The certificate should be in PEM format without BEGIN and END tags. Only used if OCSP Responder URI is set.");
```

Comment:
Is it possible to specify more clearly that this is not mandatory field? For example add another sentence to help text with something like "If this is not set, then the certificate of the CA, who issued the client certificates is used to verify the signature of OCSP response."